build/dependencies.yaml

```diff
@@ -85,32 +85,43 @@ dependencies:
 
   # Golang
   - name: "golang: upstream version"
-    version: 1.16.7
+    version: 1.17
     refPaths:
     - path: build/build-image/cross/VERSION
     - path: cluster/addons/fluentd-elasticsearch/es-image/Dockerfile
       match: 'FROM golang\:\d+.\d+(alpha|beta|rc)?\.?(\d+)?'
-    - path: test/images/Makefile
-      match: GOLANG_VERSION=\d+.\d+(alpha|beta|rc)?\.?\d+
     - path: staging/publishing/rules.yaml
       match: 'default-go-version\: \d+.\d+(alpha|beta|rc)?\.?(\d+)?'
-    - path: cluster/images/etcd/Makefile
-      match: 'GOLANG_VERSION\?=\d+.\d+(alpha|beta|rc)?\.?(\d+)?'
+    - path: test/images/Makefile
+      match: GOLANG_VERSION=\d+.\d+(alpha|beta|rc)?\.?\d+
 
-  # Golang pre-releases are denoted as `1.y<pre-release stage>.z`
+  # TODO(releng): Bump after:
+  #               1. go1.17 is released
+  #               2. initial go1.17 PR has merged
+  - name: "golang: minimum supported version"
+    version: 1.16.0
```

Comment:
go ahead and drop this stanza... we want CI to require go1.17 before this PR merges